src/guide/jbs-jdk-bug-system.md

```diff
@@ -345,7 +345,17 @@ This table contains some frequently used JBS labels and their meaning. Please he
   <tr>
     <td class="dictionary">[*(Rel)*[-na]{.jbs-label}]{#rel-na}</td>
     <td class="dictionary">
-      The [Affects Version/s]{.jbs-field} field is used to indicate the releases where an issue has been seen - it's implied that the issue is also applicable to newer releases. Where this isn't the case - for instance a bug in code that was removed in *(Rel)* - then use the *(Rel)-na* to indicate this. Note that there should only be **one** *(Rel)*[-na]{.jbs-label} label on any JBS issue.
+      Labels of the form [11-na]{.jbs-label} or [21-na]{.jbs-label} should be used when a bug is not applicable to a **more recent** release family.
+
+      The [Affects Version/s]{.jbs-field} field is used to indicate the releases where an issue has been seen - it's implied that the issue is also applicable to newer releases. Where this isn't the case - for instance a bug in code that was removed in *(Rel)* - then use *(Rel)*[-na]{.jbs-label} to indicate this. Note that there should only be **one** *(Rel)*[-na]{.jbs-label} label on any JBS issue.
```

Comment:
Take the case documented above where a bug is found in 11.x and we ask they check if it affects mainline. If it doesn't then they add, e.g. `23-na`, but this tells us nothing about when it was actually fixed or disappeared. If someone checks 21 and find the bug is not there then shouldn't `21-na` also be added? Or should it replace `23-na` - with the assumption that `<rel>-na` implies `<rel+N>-na` for all `N`?

Reply:
The label should be updated (replaced) in that case. I have tried to clarify this in the text.